File: HappyDog.WindowsUI/Services/Configuration.cs
using HappyDog.WindowsUI.Models;
using System.Collections.Generic;

namespace HappyDog.WindowsUI.Services
{
    public static class Configuration
    {
        public static List<Category> Categories => new List<Category>
        {
            new Category { Id = 1, Label = ".Net", Value = ".net", Color = "#7014e8" },
            new Category { Id = 2, Label = "数据库", Value = "db", Color = "#f65314" },
            new Category { Id = 3, Label = "Windows", Value = "windows", Color = "#00a1f1" },
            new Category { Id = 4, Label = "阅读", Value = "read", Color = "#7cbb00" },
            new Category { Id = 5, Label = "随笔", Value = "essays", Color = "#ffbb00" }
        };
    }
}

File: HappyDog.Angular/src/app/data/configuration.ts
import { Category } from "../models/category";

export class Configuration {
  public static categories: Category[] = [
    { id: 1, value: 'net', label: '.Net', color: '#7014e8', iconClass: 'fa fa-cubes' },
    { id: 2, value: 'db', label: '数据库', color: '#f65314', iconClass: 'fa fa-database' },
    { id: 3, value: 'windows', label: 'Windows', color: '#00a1f1', iconClass: 'fa fa-windows' },
    { id: 4, value: 'read', label: '阅读', color: '#7cbb00', iconClass: 'fa fa-bookmark' },
    { id: 5, value: 'essays', label: '随笔', color: '#ffbb00', iconClass: 'fa fa-pencil' },
  ]
}
